main/swadge2024.c

```diff
@@ -157,7 +157,6 @@
     #define RTC_DATA_ATTR
 #endif
 
-#define EXIT_TIME_US          1000000
 #define DEFAULT_FRAME_RATE_US 40000
```

Comment:
Hah, I moved {{DEFAULT_FRAME_RATE_US}} to the header too. You're gonna want to `git merge origin/main` here.

Reply:
Heh